src/mbgl/map/transform_state.hpp

```diff
@@ -81,7 +81,7 @@ class TransformState {
 
     // Limit the amount of zooming possible on the map.
     double min_scale = std::pow(2, 0);
-    double max_scale = std::pow(2, 20);
+    double max_scale = std::pow(2, 22);
```

Comment:
See my comment [below](https://github.com/mapbox/mapbox-gl-native/pull/5418#issuecomment-228987394) and also https://github.com/mapbox/mapbox-gl-native/issues/4272.